configuration/cdap-site.xml

```diff
@@ -777,168 +674,292 @@
+    <value>1440</value>
+    <description>Log cleanup interval in minutes</description>
+  </property>
+
+  <property>
+    <name>log.kafka.topic</name>
+    <value>logs.user-v2</value>
     <description>
-      CDAP Router service bind port for HTTPS
+      Kafka topic name used to publish logs
     </description>
   </property>
 
   <property>
-    <name>router.ssl.server.port</name>
-    <value>${router.ssl.bind.port}</value>
-    <final>true</final>
+    <name>log.publish.num.partitions</name>
+    <value>10</value>
+    <description>Number of CDAP Kafka service partitions to publish the logs to</description>
   </property>
 
-  -->
-
-  <!-- Kerberos -->
-
   <property>
-    <name>cdap.master.kerberos.keytab</name>
-    <value></value>
+    <name>log.retention.duration.days</name>
+    <value>7</value>
     <description>
-      The full path to the Kerberos keytab file containing the CDAP Master service's
-      credentials
+      Log file HDFS retention duration in days
     </description>
   </property>
 
   <property>
-    <name>cdap.master.kerberos.principal</name>
-    <value></value>
+    <name>log.saver.max.instances</name>
+    <value>${master.service.max.instances}</value>
     <description>
-      Example: "CDAP_PRINCIPAL/_HOST@EXAMPLE.COM". The Kerberos primary user
-      that should be used to login to the CDAP Master service. Substitute
-      the Kerberos primary (user) for CDAP_PRINCIPAL, and your domain for
-      EXAMPLE.COM. The string "_HOST" will be substituted with the local
-      hostname.
+      Maximum number of log saver instances to run in YARN
     </description>
   </property>
 
   <property>
-    <name>kerberos.auth.enabled</name>
-    <value>false</value>
+    <name>log.saver.num.instances</name>
+    <value>1</value>
+    <description>Number of log saver instances to run in YARN</description>
+    <value-attributes>
+      <type>int</type>
+      <minimum>1</minimum>
+      <overridable>false</overridable>
+    </value-attributes>
+  </property>
+
+  <property>
+    <name>log.saver.run.memory.megs</name>
+    <value>1024</value>
+    <description>Memory in megabytes allocated for log saver instances to run in YARN</description>
+    <value-attributes>
+      <type>int</type>
+      <minimum>0</minimum>
+      <maximum>268435456</maximum>
+      <unit>MB</unit>
+      <increment-step>256</increment-step>
+      <overridable>false</overridable>
+    </value-attributes>
+  </property>
+
+  <property>
+    <name>log.saver.run.num.cores</name>
+    <value>2</value>
     <description>
-      Determines if Kerberos authentication is enabled
+      Number of cores for each log saver instance in YARN
+    </description>
+    <display-name>Log Saver YARN CPU Cores</display-name>
+    <value-attributes>
+      <type>int</type>
+      <minimum>1</minimum>
+      <maximum>4</maximum>
+      <increment-step>1</increment-step>
+      <overridable>false</overridable>
+    </value-attributes>
+  </property>
+
```

Comment:
That's on purpose. It's a YARN container. Everything that runs in YARN *needs* either `0.0.0.0` or the IP address of the NodeManager it's running within... since we can't know that ahead of time, the only "valid" value is `0.0.0.0` which comes from `cdap-default.xml` in CDAP.